lighthouse-core/test/audits/metrics-test.js

```diff
@@ -45,4 +48,19 @@ describe('Performance: metrics', () => {
     const result = await MetricsAudit.audit(artifacts, context);
     expect(result.details.items[0].interactive).toEqual(undefined);
   });
+
+  it('evaluates valid input (with lcp) correctly', async () => {
+    const artifacts = {
+      traces: {
+        [MetricsAudit.DEFAULT_PASS]: lcpTrace,
+      },
+      devtoolsLogs: {
+        [MetricsAudit.DEFAULT_PASS]: lcpDevtoolsLog,
+      },
+    };
+
+    const context = {settings: {throttlingMethod: 'devtools'}, computedCache: new Map()};
+    const result = await MetricsAudit.audit(artifacts, context);
+    expect(result.details.items[0]).toMatchSnapshot();
```

Comment:
FWIW @connorjclark most of the metric snapshots are because of lantern dependencies where the expected value is a moving target as we make improvements. in cases like this where the right answer is never expected to change we could just hardcode the expect.

Reply:
that's fair, but the other test in this file leverages the snapshotting utility.